packages/runtime-dom/src/modules/events.ts

```diff
@@ -1,4 +1,4 @@
-import { hyphenate, isArray } from '@vue/shared'
+import { hyphenate, isArray, isFF } from '@vue/shared'
```

Comment:
since `isFF` isn't actually used anywhere else, and it being very version specific, let's just keep the detection logic here instead of in `@vue/shared`. Can also directly a variable named `bailTimestampCheck`.

Reply:
@yyx990803 Firefox isn't the only platform that is affected by this. [See this 2.6 PR](https://github.com/vuejs/vue/pull/10459) for more.